exercises/exercise_5/main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import "fmt"
+
+type Node struct {
+	Value int
+	Next  *Node
+}
+
+func Prepend(head **Node, x int) {
+	*head = &Node{x, *head}
+}
+
+func Append(head **Node, x int) {
+	if *head == nil {
```

Comment:
nit: consider simplifying the code a bit, both cases could be handled in exactly the same way (a fun of double pointers)